RibonDAO/app
# feat: new profile section

# Description

- Adds the new profile header

## Did you use styled-components?


## Type of change

Please delete options that are not relevant.


## How to test

- Login with a member and non member user to see the new profile page
- Go to the club screen to see the new profile badge

## Images 

<p float="left">
<img src="https://github.com/RibonDAO/app/assets/57116106/2b03a43f-7aed-4d7f-bd8b-b406cbf84f89" />
<img src="https://github.com/RibonDAO/app/assets/57116106/bd89309d-b592-470b-bba2-6a0a95b90e14" />
</p>
